Resources/Prototypes/_Goobstation/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -0,0 +1,89 @@
+- type: entity
+  parent: PosterBase
+  id: PosterLegitBigBrotherAI
+  name: "AI Sees All"
+  description: "A poster that reminds you that your secrets are not hidden from your Station AI"
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)